src/data/markdown/translated-guides/en/04 Results visualization/08 StatsD.md

```diff
@@ -21,3 +21,5 @@ The following options can be configured:
 | `K6_STATSD_NAMESPACE`     | The namespace used as a prefix for all the metric names. The default value is `k6`.                    |
 | `K6_STATSD_PUSH_INTERVAL` | Configure how often data batches are sent. The default value is `1s`.                                  |
 | `K6_STATSD_BUFFER_SIZE`   | The buffer size. The default value is `20`.                                                            |
+| `K6_STATSD_ENABLE_TAGS`   | If `true` enables sending tags. `false` by default as old versions of statsd, prior to v0.9.0 did not support tags. |
+| `K6_STATSD_TAG_BLOCKLIST` | This is a comma-separated list of tags that should NOT be sent to statsd. For example, "tag1, tag2". The default value is empty. |
```

Comment:
Comma separated, not comma-and-space, right?

```suggestion
| `K6_STATSD_TAG_BLOCKLIST` | This is a comma-separated list of tags that should NOT be sent to statsd. For example, "tag1,tag2". The default value is empty. |
```

Reply:
it technically works with spaces as well ;)